Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -318,46 +319,63 @@
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If a Ghost chases you, go Out of Bounds to reset its spooky timer! It will not go after you for 10-20 seconds after this event.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're bad at this game, the Out of Bounds special taunt messages will make fun of you. This can be disabled in the Options.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest is open source! Check out https://github.com/The-New-Platinum-Team/PlatinumQuest-Dev for more information!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: YOUR custom levels can be included in the game! Check out the Marble Blast Community Master Sheet linked in our official Discord server for more information!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Pressing <func:bind forceRespawn> will respawn you at the last checkpoint. Holding <func:bind forceRespawn> for one second will restart the whole level from the beginning.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Did you know you can play Marble Blast Platinum in your browser? Try out https://marbleblast.vaniverse.io/ or https://marbleblast.randomityguy.me/!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Marble It Up!' on Steam!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Great Marble Adventure' on Steam!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Custom Levels' in the Community section of the Main Menu! You'll be able to download and play custom levels all from ingame!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Want to upload your custom levels for people to play? Head to https://marbleland.vaniverse.io/!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: You'll gain more height from a Super Jump, Blast, or wall hit by using them immediately after jumping, rather than at the peak of your jump.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're bad at this game, the Out of Bounds special taunt messages will make fun of you.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest is open source! Check out github.com/The-New-Platinum-Team/PlatinumQuest-Dev for more information!"; //Yoshi
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: YOUR custom levels can be included in the game! Check out the Marble Blast Community Master Sheet linked in our official Discord server for more information!"; //Yoshi
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Pressing <func:bind forceRespawn> will respawn you at the last checkpoint. Holding <func:bind forceRespawn> for one second will restart the whole level from the beginning. In Multiplayer, this is increased to 3 seconds, and only works as the server host.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Did you know you can play Marble Blast Platinum in your browser? Try out marbleblast.vaniverse.io or marbleblast.randomityguy.me!"; //Yoshi
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Check out 'Marble It Up!' and 'Marble It Up! Ultra' on Steam!"; //Yoshi
```

Comment:
How about `Tip: Check out 'Marble It Up!' and 'Marble It Up! Ultra' on PC and consoles!`

Reply:
If someone's playing PQ, they have a computer capable of playing games, and Steam is the best version of MIW anyway since it's the only one with custom levels

Reply:
Steam is the best way to play on PC, but I'm sure there are plenty of people are playing PQ on old crappy laptops but they have a switch/their family has an xbox/etc. `PC and consoles` covers more cases